msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md

```diff
@@ -0,0 +1,459 @@
+
+// Change the selected tool to eraser
+document.getElementById("eraser").onclick = () => {
+  inkingManager.tool = InkingTool.eraser;
+};
+// Increase the tip size for eraser
+document.getElementById("eraser-size").onclick = () => {
+  inkingManager.eraserSize = inkingManager.eraserSize + 1;
+};
+```
+
+#### Point eraser tool
+
+:::image type="content" source="../assets/images/teams-live-share/canvas-point-eraser-tool.gif" alt-text="Teams Live Share canvas point eraser tool":::
+
+The point eraser tool erases individual points within strokes that cross its path by splitting existing strokes in half. This tool is computationally expensive and may result in slower frame rates for your users.
+
+> [!NOTE]
+> The point eraser shares the same eraser point size as the regular eraser tool.
+
+```html
+<div>
+  <button id="point-eraser">Enable Point Eraser</button><br />
+</div>
+```
+
+```javascript
+import {
+  InkingManager,
+  InkingTool,
+} from "@microsoft/live-share-canvas";
+// ...
+
+// Change the selected tool to eraser
+document.getElementById("point-eraser").onclick = () => {
+  inkingManager.tool = InkingTool.pointEraser;
+};
+```
+
+#### Laser pointer
+
+:::image type="content" source="../assets/images/teams-live-share/canvas-laser-tool.gif" alt-text="Teams Live Share canvas laser pointer tool":::
+
+The laser pointer is unique because the tip of the laser has a trailing effect as you move your mouse. When you draw strokes, they will render for a short period of time before they fade out completely. This tool is perfect for pointing out information on the screen during a meeting, as the presenter doesn't have to switch between tools to erase strokes.
+
+```html
+<div>
+  <button id="laser">Enable Laser Pointer</button><br />
+  <label for="laser-color">Select a color:</label>
+  <input type="color" id="laser-color" name="laser-color" value="#000000" />
+  <button id="laser-tip-size">Increase tip size</button>
+</div>
+```
+
+```javascript
+import {
+  InkingManager,
+  InkingTool,
+  fromCssColor,
+} from "@microsoft/live-share-canvas";
+// ...
+
+// Change the selected tool to laser pointer
+document.getElementById("laser").onclick = () => {
+  inkingManager.tool = InkingTool.highlighter;
+};
+// Change the selected color for laser pointer
+document.getElementById("laser-color").onchange = () => {
+  const colorPicker = document.getElementById("laser-color");
+  inkingManager.laserPointerBrush.color = fromCssColor(colorPicker.value);
+};
+// Increase the tip size for laser pointer
+document.getElementById("laser-tip-size").onclick = () => {
+  inkingManager.laserPointerBrush.tipSize = inkingManager.laserPointerBrush.tipSize + 1;
+};
+```
+
+#### Line and arrow tools
+
+:::image type="content" source="../assets/images/teams-live-share/canvas-line-tool.gif" alt-text="Teams Live Share canvas line tool":::
+
+The line tool allows users to draw straight lines from one point to another, with an optional arrow that can be applied to the end.
+
```

Comment:
Do you think we should add a comment about pressing shift will force a 90degre or 180 degree line?

Reply:
Probably, but then I have to go into all of the keyboard shortcuts in a table. I'll see if I find time.